app/models/inventory.rb

```diff
@@ -6,13 +6,38 @@
 #  name        :string           not null
 #  deadline    :datetime         not null
 #  district_id :integer          not null
+#  owner_id    :integer
 #
 
 class Inventory < ActiveRecord::Base
+  include Authority::Abilities
```

Comment:
I skimmed through the documentation on Authority and I feel pretty good about it, but it's not immediately apparent that we directly call any of its methods for an `Inventory` model.  Is it the case that this is done transparently at some layer?